build/Dockerfile.kip-aws-meteringagent

```diff
@@ -1,7 +1,8 @@
-FROM 689494258501.dkr.ecr.us-east-1.amazonaws.com/elotl/kip-aws-meteringagent:v1.0.1 AS base
+FROM 689494258501.dkr.ecr.us-east-1.amazonaws.com/elotl-dev/kip-aws-meteringagent:v1.0.3 AS base
 
 FROM amazonlinux
 
+ENV PRODUCT_CODE=9iyga48767smc3nbdqo31qq5c
```

Comment:
Good to move this back over here. We had to update the product code but, to get a product code, it looked like we needed to push a new container.  I moved the product code into the manifest to deal with that.